src/WebJobs.Script.WebHost/Web.config

```diff
@@ -35,7 +35,7 @@
           <clear />
         </namespaces>
       </pages>
-      <httpRuntime targetFramework="4.5" maxRequestLength="102400" maxUrlLength="8192" maxQueryStringLength="4096" requestPathInvalidCharacters="" enableVersionHeader="false"/>
+      <httpRuntime targetFramework="4.7.1" maxRequestLength="102400" maxUrlLength="8192" maxQueryStringLength="4096" requestPathInvalidCharacters="" enableVersionHeader="false"/>

```

Comment:
Above I see `<compilation debug="true" targetFramework="4.6.1">`. Do we need to change that as well?

Reply:
Note that in my patch release, I set it to `4.7` and not `4.7.1`. It probably doesn't matter, but if we want to be on the safe side, maybe we should stick to `4.7`, since we're getting a bunch of coverage in Prod.

For `compilation`, that may affect the compilation of views, and I don't think we even have any, so it's probably irrelevant.

Reply:
Wouldn't really impact us, but updated for consistency.

Reply:
Setting to 4.7.1 to actually match the framework. If there are no oppositions, it would be nice to stick with that. I don't expect that to cause any issues.

Reply:
That's fine. As long as we deploy the next build slowly, we'll catch any regression (which is super unlikely). But for sanity, please do quick check that it still solves the two issues at stake with that value (just in case!).

Reply:
Validated that this addresses the issue we were dealing with.